charts/rancher-monitoring/v0.2.2/questions.yml

```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.5.8-rc1
+rancher_max_version: 2.5.8
```

Comment:
Code looks good, but few more things: 
* based on convo earlier we only need to do mirroring in 2.6, so I can move this to point at dev-v2.6 branch instead
* Do we need to adjust the other chart like arvind said ? 
* Lets squash to only 2 commits, 1 copy and 1 for changes

Reply:
> Do we need to adjust the other chart like arvind said ?

Do you mean the v0.1.x chart that Arvind mentioned [here](https://github.com/rancher/system-charts/pull/459#pullrequestreview-657235561)?  I think we do not need to do it since we will only release the new monitoring chart in the monitoring 0.2.x line in Rancher v2.6

Reply:
Ok so 2.6 only, no old chart. I will change base branch, just need to squash commits.